pyccl/halos/profiles/profile_base.py

```diff
@@ -202,13 +165,12 @@ def fourier(self, cosmo, k, M, a, mass_def):
             are scalars, the corresponding dimension will be
             squeezed out on output.
         """
-        if getattr(self, '_fourier', None):
-            f_k = self._fourier(cosmo, k, M, a, mass_def)
-        elif getattr(self, '_real', None):
-            f_k = self._fftlog_wrap(cosmo, k, M, a, mass_def, fourier_out=True)
-        return f_k
+        if getattr(self, "_fourier", None):
+            return self._fourier(cosmo, k, M, a, mass_def)
+        return self._fftlog_wrap(cosmo, k, M, a, mass_def, fourier_out=True)
 
-    def projected(self, cosmo, r_t, M, a, mass_def):
+    @warn_api(pairs=[("r_t", "r")])
+    def projected(self, cosmo, r, M, a, *, mass_def=None):
```

Comment:
`r_t` seems like a perfectly fine variable name. Why change it?

Reply:
(it's the transverse radius)

Reply:
The docs don't differentiate between this r and the r in real, fourier etc. So, either we name everything `r`, or we clarify it in the docs that they represent different quantities.

Reply:
I would rather update the docs. This variable name is actually fine, and changing it will only make people's lives (even) more complicated when changing to v3

Reply:
done